Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go

```diff
@@ -1,12 +1,19 @@
 package azure
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/sha1"
+	"encoding/base64"
+	"fmt"
 	"net/http"
 	"net/url"
 	"strconv"
 	"time"
 
 	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
+	"github.com/dgrijalva/jwt-go"
```

Comment:
go-check is a test package that's why that was the case as far as I
remember. When someone vendors our SDK I don't see why they should get go
check. They're not going to test the package.

also godep leaves that pkg out, again because it's referenced only in test
builds; not the normal builds.
On Tue, Nov 10, 2015 at 11:07 AM Cole Mickens <notifications@github.com>
wrote:

> In
> Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure/token.go
> <https://github.com/Azure/azure-sdk-for-go/pull/233#discussion_r44449656>:
>
> >  	"net/http"
> >  	"net/url"
> >  	"strconv"
> >  	"time"
> >
> >  	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest"
> > +	"github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12"
> > +	"github.com/dgrijalva/jwt-go"
>
> I've added a new commit; it vendors sub-dependencies (pkcs12, jwt-go, and
> check.v1), includes the sub-dependencies' test code (keeping in line with
> how autorest was originally vendored), and rewrites paths.
>
> Hopefully this resolves any concerns.
>
> LMK and I'll squash it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-sdk-for-go/pull/233/files#r44449656>.
>
-- 
—Ahmet Alp Balkan